setup_test_env.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+
+
+# install chrome
+wget -nc https://dl-ssl.google.com/linux/linux_signing_key.pub 
+cat linux_signing_key.pub | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/linux_signing_key.gpg  >/dev/null 
+sudo sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/chrome.list' 
+sudo apt update 
+sudo apt install google-chrome-stable 
+
+
+# pyodide build and test
+pip install pytest 
+pip install pyodide-build pytest-pyodide 
+# install chromedriver stuff for selenium to control chrome
+pip install selenium webdriver-manager 
+pip install chromedriver-binary-auto
```

Comment:
Installing them via GH actions as in https://github.com/pyodide/micropip/blob/32117636152a253fddd041273a2c21cd00c449f1/.github/workflows/main.yml#L59 is make it probably easier to make sure the chrome and chrome-driver versions match. 

But this is already a very large improvement compared to when there were no tests. Great to see work on this package: